CSS/style.css

```diff
@@ -0,0 +1,103 @@
+@import url('https://fonts.googleapis.com/css2?family=Clicker+Script&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,900;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+  margin: 0%;
+  padding: 0%;
+  background-color: #e5e5e5;
+}
+
+.My_logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 22px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 6%;
+  padding-left: 8%;
+  padding-right: 8%;
```

Comment:
I suggest that you set height of the flexbox  so that the vertical alignment take effect and give this section same width as the next one 80% this way both will resize same way with screen size this will fix the issue mentioned by previous reviewers 
also as mentioned earlier remove all absolute positioned images as this is what's affecting your design those images should be used as background images this way they won't affect the flow of dom elements 
 **after deleting images:**
![after deleting images](https://user-images.githubusercontent.com/59707859/219730244-ddd6ecb7-de2b-47c5-a2d2-e28de32b576f.PNG)

**your design:**
![responsiveness issues](https://user-images.githubusercontent.com/59707859/219730255-617b7c42-365c-476d-81d9-ab29e679fe33.PNG)

Reply:
> I suggest that you set height of the flexbox so that the vertical alignment take effect and give this section same width as the next one 80% this way both will resize same way with screen size this will fix the issue mentioned by previous reviewers also as mentioned earlier remove all absolute positioned images as this is what's affecting your design those images should be used as background images this way they won't affect the flow of dom elements **after deleting images:** ![after deleting images](https://user-images.githubusercontent.com/59707859/219730244-ddd6ecb7-de2b-47c5-a2d2-e28de32b576f.PNG)
> 
> **your design:** ![responsiveness issues](https://user-images.githubusercontent.com/59707859/219730255-617b7c42-365c-476d-81d9-ab29e679fe33.PNG)

What Can I do


I try it So Many Times. would any one help me.

Reply:
> I suggest that you set height of the flexbox so that the vertical alignment take effect and give this section same width as the next one 80% this way both will resize same way with screen size this will fix the issue mentioned by previous reviewers also as mentioned earlier remove all absolute positioned images as this is what's affecting your design those images should be used as background images this way they won't affect the flow of dom elements **after deleting images:** ![after deleting images](https://user-images.githubusercontent.com/59707859/219730244-ddd6ecb7-de2b-47c5-a2d2-e28de32b576f.PNG)
> 
> **your design:** ![responsiveness issues](https://user-images.githubusercontent.com/59707859/219730255-617b7c42-365c-476d-81d9-ab29e679fe33.PNG)

I can't Proceed,  So I need help from any one of you???????

Reply:
> I suggest that you set height of the flexbox so that the vertical alignment take effect and give this section same width as the next one 80% this way both will resize same way with screen size this will fix the issue mentioned by previous reviewers also as mentioned earlier remove all absolute positioned images as this is what's affecting your design those images should be used as background images this way they won't affect the flow of dom elements **after deleting images:** ![after deleting images](https://user-images.githubusercontent.com/59707859/219730244-ddd6ecb7-de2b-47c5-a2d2-e28de32b576f.PNG)
> 
> **your design:** ![responsiveness issues](https://user-images.githubusercontent.com/59707859/219730255-617b7c42-365c-476d-81d9-ab29e679fe33.PNG)

Some of the reviewer told me before add a background image, I add it, but you said again delete the background image?????......   if I delete the background image it does not look like the figma template.